charts/assemble-backstage/templates/postgres-ss.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if not .Values.postgres.external }}
```

Comment:
how does `postgres.external` play in? I do not see any considerations in the location of the psql server in the config?

Reply:
Added config for specifying an external host